internal/backends/linuxkms/calloop_backend.rs

```diff
@@ -162,7 +162,7 @@ impl i_slint_core::platform::Platform for Backend {
                 .seat
                 .borrow_mut()
                 .open_device(&device)
-                .map_err(|e| format!("Error opening device: {e}"))?;
+                .map_err(|e| format!("Error opening device {device}: {e}"))?;
```

Comment:
yay for more better error that are more meaningful to end user

Reply:
Urgh, I pushed the wrong change into the wrong branch :)

Reply:
Would have been better if it compiled :-)

Reply:
Let’s not get caught up in details ;-)